test/es-module/test-esm-import-meta-resolve.mjs

```diff
@@ -106,3 +106,14 @@ await assert.rejects(import('data:text/javascript,export default import.meta.res
     stdout: 'http://example.com/\n',
   });
 }
+
+{
+  const result = await spawnPromisified(execPath, [
+    '--no-warnings',
+    '--experimental-import-meta-resolve',
+    path('import-meta-resolve-loader.js'),
```

Comment:
Can we avoid creating a fixtures file, and also `path` is arguably not a very good name as I would expect it to refer to `node:path`
```suggestion
    '--eval',
    'import.meta.resolve("foo", "http://example.com/bar.js")',
```